ddl/db_test.go

```diff
@@ -582,9 +582,20 @@ func (s *testDBSuite) TestAddMultiColumnsIndex(c *C) {
 }
 
 func (s *testDBSuite) TestAddIndex(c *C) {
+	s.testAddIndex(c, false, "create table test_add_index (c1 bigint, c2 bigint, c3 bigint, primary key(c1))")
+	s.testAddIndex(c, true, `create table test_add_index (c1 bigint, c2 bigint, c3 bigint, primary key(c1))
+			      partition by range (c2) (
+			      partition p1 values less than (61440),
+			      partition p2 values less than (122880),
+			      partition p3 values less than (204800),
+			      partition p4 values less than maxvalue)`)
```

Comment:
Please change the `less than` value to make inserted data into the different partitions.

Reply:
It should be in different partitions:
```
n := base + i*defaultBatchSize + i
rand.Intn(n)
```
i = 1 .. 100
base = 2048
defaultBatchSize = 2048
https://github.com/pingcap/tidb/pull/6814/files#diff-a10ae18073ce5ff6942daadcaf8afc59R614

Reply:
Got it.